rosbot_utils/package.xml

```diff
@@ -4,21 +4,21 @@
   <name>rosbot_utils</name>
   <version>0.13.0</version>
   <description>Utilities for ROSbot 2R and 2 PRO</description>
-  <license>Apache License 2.0</license>
-
-  <author email="dominik.nwk@gmail.com">Dominik Nowak</author>
```

Comment:
Dominik really is the author of this package. Please don't skip him

Reply:
Add as mainainer